### DISCUSSION TITLE
I'm experimenting Question Bypass and Easy Mode.

### DISCUSSION THREAD

**Discussion by NX721** [Prodigy-Hacking/ProdigyMathGameHacking#803] [(A) Anything]
Finding Question Bypass and Easy Mode.

**Comment by PatheticMustan** [comment]
ok????

**Reply by NX721** [reply]
Ok then lol

**Reply by ghost** [reply]
can't you just win the battle instead of doing easy mode?

**Reply by ghost** [reply]
or just make a grade 1 account because a person who found this is smarter than a 4 year old
